### ISSUE TITLE
Design problem, I need some help

### ISSUE THREAD

**Issue by ChrisSka** [novastream/Modern-Theme#10]
I have a design problem with the status display in the Kanban view and in the detail view.
I can't get any further somehow.
I can't get rid of these white gaps and I don't know exactly where to look for them.

Kanban view:
![image](https://github.com/user-attachments/assets/8b100c7a-4c8b-4ec0-aa3a-62faac058cea)

and this is what it should actually look like: (detailed view)
![image](https://github.com/user-attachments/assets/97f2bad1-d349-494f-a96f-9047a69c4448)

**Comment by novastream** [comment]
Hi Chris!

This is a known issue and just haven't had the time to fix it just yet. I will try to get it fixed this weekend :)

**Comment by ChrisSka** [comment]
Hy,
did you have time to find a solution?

**Comment by novastream** [comment]
Hi, sorry for the delay. Should be fixed now in the latest commit :)

**Comment by ChrisSka** [comment]
Hmmm... somehow that has no effect on me at all. Nothing changes at all.

What's strange is that it looks different for the entities.
Sometimes a small field is white, then sometimes larger.

![image](https://github.com/user-attachments/assets/0491431e-4739-430e-9a40-ee543c5f1d29)

![image](https://github.com/user-attachments/assets/890e48d3-a424-44fb-81ad-5a818e90a52d)

Here I have 2 different white fields:
![image](https://github.com/user-attachments/assets/dcc24a6a-66a2-4133-94d7-a947c0364558)

**Comment by novastream** [comment]
Yeah, found out it only worked in my older dev version. Will take another go at this.

**Comment by novastream** [comment]
So the markup has changed and the arrows has been removed in the original themes so I've also removed them from this theme and raised the minimum espo version in the manifest. May look into the arrow styling (if possible without altering the markup) in the feature.

**Comment by ChrisSka** [comment]
This is a shame, as the standard EspoCRM simply looks boring.

The only thing that irritates me is that it works in the detail view and unfortunately not in the Kanban view.
I'll have to have a look at it from the extension, as it is displayed in the detail view from an extension.

**Comment by novastream** [comment]
Agreed, the "previous" class got removed from the elements so I can't use the default css to creates the arrows. I'm sure it's possible to style them in another way but it will take some time. I'll look into it but thought I'd remove it all together first. I'll keep this issue open.

**Comment by ChrisSka** [comment]
![image](https://github.com/user-attachments/assets/0e87f251-5bf8-4ac9-afa5-0227d35e1b3b)

I've already made a little progress, so that the arrows are at least getting closer ;-)
But not perfect yet, unfortunately....

**Comment by novastream** [comment]
Looking good, think you have to utilize :after selectors or negative positioning to get the colors to overlap

**Comment by ChrisSka** [comment]
Hy, I have now decided to change it to the EspoCRM standard after all, i.e. without these arrows.
Maybe the arrows will come back at some point.

**Comment by novastream** [comment]
Closing this issue, with the new markup its quite hard to change the styling back to what it was pre v9.
